test/skimNtuple2018_HHbtag.cpp

```diff
@@ -125,46 +125,47 @@ const float DYscale_MTT_vHighPt_err[3] = {0.0046517215, 0.11443309   , 0.0645193
+    {0.661713657124 , 0.26243493199 , 0.198163975706 , 0.0916283420008 , 0.0351465223751 , 0.00911500115122 , 0.00416132811439 , 0.000936737452394 , 0.0},
+    {0.668346540329 , 0.261732816396 , 0.197497854085 , 0.0885734561017 , 0.0411219666886 , 0.00920504582394 , 0.00418420652186 , 0.000938040286125 , 0.0},
     {0.0 , 0.0 , 0.0 , 0.0 , 0.0 , 0.0 , 0.0 , 0.0 , 0.0},
     {0.0 , 0.0 , 0.0 , 0.0 , 0.0 , 0.0 , 0.0 , 0.0 , 0.0},
     {0.0 , 0.0 , 0.0 , 0.0 , 0.0 , 0.0 , 0.0 , 0.0 , 0.0}
   },
   {
-    {0.580157109833 , 0.323048584481 , 0.316235063773 , 0.155031902758 , 0.0166252750008 , 0.00929109993408 , 0.0119071053391 , 0.0155571504236 , 0.000294085695839},
-    {0.576159635871 , 0.32225013185 , 0.317509585462 , 0.156832157172 , 0.0176872450509 , 0.00959797340356 , 0.0119359304201 , 0.0116739568912 , 0.000294219655312},
-    {0.114285902101 , 0.0983303196122 , 0.0960268363933 , 0.0716322046668 , 0.0160449241071 , 0.00911109415307 , 0.0113069937242 , 0.0141381182245 , 0.000294219655312},
+    {0.538419462951 , 0.238551569309 , 0.182617431561 , 0.0887633713842 , 0.0380851630896 , 0.00861460228774 , 0.0034763580742 , 0.00093649304071 , 4.5558708978e-05},
+    {0.543364625407 , 0.239497565938 , 0.185414970409 , 0.0916754209195 , 0.0397221792446 , 0.0090682313946 , 0.00416056693699 , 0.000935556069265 , 4.55600433279e-05},
+    {0.544414477224 , 0.240860536713 , 0.183153728781 , 0.0893258021275 , 0.0393245639366 , 0.00903754894581 , 0.00416735510614 , 0.000935996625371 , 4.55600433279e-05},
     {0.0 , 0.0 , 0.0 , 0.0 , 0.0 , 0.0 , 0.0 , 0.0 , 0.0},
     {0.0 , 0.0 , 0.0 , 0.0 , 0.0 , 0.0 , 0.0 , 0.0 , 0.0}
   },
   {
-    {0.682278722716 , 0.362765385278 , 0.352621143228 , 0.165382670894 , 0.0174462106614 , 0.00958560178972 , 0.0113611990133 , 0.0155822731809 , 0.00029419199659},
-    {0.658904274958 , 0.355925643213 , 0.346908775498 , 0.160894989801 , 0.0172806075072 , 0.00961908326477 , 0.0118871488334 , 0.015698805782 , 0.000294219655312},
-    {0.116329810304 , 0.103266407792 , 0.101348794702 , 0.0755949912983 , 0.0146934201983 , 0.00901755115964 , 0.0110541461103 , 0.0145892891019 , 0.000294219655312},
-    {0.11037555227 , 0.0943817218953 , 0.0935595693025 , 0.0707927004997 , 0.0158861665046 , 0.00910876641137 , 0.0109700992432 , 0.0 , 0.0},
+    {2.90732304488 , 0.380710347472 , 0.258349388757 , 0.105264839118 , 0.0414475737267 , 0.00888246109324 , 0.00428097941008 , 0.000829691970354 , 4.55596260999e-05},
+    {2.90732304488 , 0.382514374571 , 0.25566379225 , 0.104069449314 , 0.039544326655 , 0.0104613619203 , 0.00417876415726 , 0.000937512258944 , 4.55600433279e-05},
+    {2.90732304488 , 0.379261342201 , 0.257502148794 , 0.10579529803 , 0.0452388596563 , 0.00917793916991 , 0.00417797465512 , 0.00093746249722 , 4.55600433279e-05},
+    {2.90732304488 , 0.387445899166 , 0.265902194159 , 0.104486477816 , 0.0418654170312 , 0.00920504582394 , 0.00418420652186 , 0.000938040286125 , 4.55600433279e-05},
```

Comment:
Just a comment for future reference: the weights in the first column are larger because of the missing 3Jet samples, the stitching is working fine.

Reply:
I'll just add that comment outside the review so that it appears more evidently